lib/web/apiserver.go

```diff
@@ -2496,9 +2496,17 @@ func makeUnifiedResourceRequest(r *http.Request) (*proto.ListUnifiedResourcesReq
 		}
 	}
 
+	var pinnedResourcesIDs []string
+	for _, id := range values["pinnedResourcesIDs"] {
+		if id != "" {
+			pinnedResourcesIDs = append(pinnedResourcesIDs, id)
+		}
+	}
```

Comment:
Yeah, I didn't even think about. I ended up refactoring this whole thing and now the IDs are pulled from user preferences in the grpc function instead of passed in the URL